pegasus/sites.v3/code.org/views/homepage_ceos_for_cs_banner.haml

```diff
@@ -0,0 +1,13 @@
+- if request.locale == "en-US" && !!DCDO.get('ceos-for-cs-banner', false)
+  = inline_css 'generated/homepage_ceos_for_cs_banner.css'
+
+  %aside.homepage-ceos-for-cs-banner
+    .wrapper
+      .top-wrapper
+        %h1 CEOs for CS
+        %a.btn{href: 'https://www.ceosforcs.com/', target: '_blank'} Read the letter
```

Comment:
nit: for links that open in a new tab, can you add `rel: "noopener noreferrer"`? There's a security vulnerability with opening links in new tabs that this mitigates. I believe it's not necessary for up to date browsers (at least [Chrome](https://chromestatus.com/feature/6140064063029248)) but good to add for older browsers

Reply:
@bethanyaconnor definitely! Just added.